core/include/moveit/task_constructor/solvers/pipeline_planner.h

```diff
@@ -58,20 +59,25 @@ class PipelinePlanner : public PlannerInterface
 	struct Specification
 	{
 		moveit::core::RobotModelConstPtr model;
-		std::string ns{ "move_group" };
+		std::string ns{ "ompl" };
```

Comment:
Hi @JafarAbdi , I'm combining  MTC with the Pilz_industrial_motion_planner pipeline in ROS2 and stumbled across this line. Is there a reason for switching from "move_group" to "ompl" at this place?